server/src/main/java/org/jboss/as/server/deployment/ManagedExplodedContentServitor.java

```diff
@@ -47,32 +49,44 @@
  * content repo to the jboss.server.data.dir.
  *
  * @author Brian Stansberry
+ * @author <a href="mailto:ropalka@redhat.com">Richard Opalka</a>
  */
-class ManagedExplodedContentServitor implements Service<VirtualFile> {
+class ManagedExplodedContentServitor implements Service {
```

Comment:
This service value is used outside of this file.

I'm not a fan of getting rid of service values, at least not for non-void services. I do not see the cost+risk/reward ratio as a good one.

Reply:
Value was not removed @bstansberry but replaced with new mechanism.
"Consumer<VirtualFile> virtualFileConsumer" is now responsible for providing the value to service consumers.
IOW line 134 replaces deleted line 159.

Reply:
Ah,ok, thanks. TBH I don't know if I ever really properly understood that before.